### PR TITLE
RELATED: BB-271 Totals with new visObj

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

File: src/components/core/tests/PureTable.spec.tsx [hunk 1]
```diff
@@ -20,16 +20,26 @@ import { ErrorStates } from '../../../constants/errorStates';
...
     oneMeasureResponse,
     oneMeasureAfm,
-    tooLargeResponse, emptyResponse
+    tooLargeResponse,
+    emptyResponse,
+    executionObjectWithTotals,
+    responseWithTotals
 } from '../../../execution/fixtures/ExecuteAfm.fixtures';
 import { AFM } from '@gooddata/typings';
+import {IIndexedTotalItem} from '../../../interfaces/Totals';
```
Comment (ivan-nejezchleb) [inline src/components/core/tests/PureTable.spec.tsx]: spaces

File: src/helpers/MdObjectHelper.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// todo-mms: Consider to implement MDObject class to encapsulate all kind of helper methods around mdObject data
+import {VisualizationObject} from '@gooddata/typings';
```
Comment (ivan-nejezchleb) [inline src/helpers/MdObjectHelper.ts]: spaces

File: src/helpers/TotalsConvertor.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import { IIndexedTotalItem } from '../interfaces/Totals';
```
Comment (ivan-nejezchleb) [inline src/helpers/TotalsConvertor.ts]: move after absolute imports

File: src/components/core/tests/PureTable.spec.tsx (hunk 1 above)
Comment (martinmosko) [reply src/components/core/tests/PureTable.spec.tsx]: done

File: src/helpers/TotalsConvertor.ts (hunk 3 above)
Comment (martinmosko) [reply src/helpers/TotalsConvertor.ts]: fixed

File: src/helpers/MdObjectHelper.ts (hunk 2 above)
Comment (martinmosko) [reply src/helpers/MdObjectHelper.ts]: fixed

**Pull request merged**